### ISSUE TITLE
[Lighthouse a11y] After selecting continue, there is a lack of error announcement when leaving M, D, Y form field blank

### ISSUE THREAD

**Issue by shreyachatterjee00** [civiform/civiform#12891]
**Describe the bug**
Issue: After selecting the "Continue" Button, there is a lack of error announcement when leaving the "Month", "Day", or "Year" Form Field blank or inputting an invalid "Date" entry.

Location: Program Qualification Form Screen 1; Comprehensive Sample Program Form Screen 1

**WCAG Success Criterion** 
4.1.3 Status Messages

**To Reproduce**
iPhone/VO
1. Properly fill in 1 or 2 of the Birthday group Form Fields Month, Day, and Year while leaving 1 or 2 Form Fields blank.
 2. Select the "Continue" Button.
 
 Or
 
 1. Properly fill the "Month" and "Year" Form Field.
 2. Input an invalid entry for "Day" e.g. 32.
 3. Select the "Continue" Button.

**Expected behavior**
 Expectation: A clear and logical error announcement should be present.

**Remediation**
Provide an error announcement with an error suggestion that is clear and logicl.
 E.g. Error "Year" is required, Error "Date" Enter a digit between 1-31
